infrastructure/web3-adapter/src/evault/evault.ts

```
@@ -0,0 +1,207 @@
+    metaEnvelope: MetaEnvelope;
+}
+
+interface StoreMetaEnvelopeResponse {
+    storeMetaEnvelope: {
+        metaEnvelope: {
+            id: string;
+            ontology: string;
+            envelopes: Array<{
+                id: string;
+                ontology: string;
+                value: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)